### PR TITLE
Implement remove edge from clustering

### PR DESCRIPTION
**Pull request by Tyler-Maclachlan**
Fix issue #21

### CODE REVIEW

**Comment by micahstubbs** [comment]
just tested this out locally at http://localhost:8080/examples/other/clusteringByZoom/


looks good!
Screen Shot 2019-07-18 at 12 33 04 AM

**Comment by micahstubbs** [comment]
ok, I logged this issue to research this new scroll-blocking warning we're getting now. https://github.com/visjs-community/visjs-network/issues/72

It's just a warning, so I think we can take this bug fix now and follow up on the warning later.

File: lib/network/modules/Clustering.js [hunk 1]
```diff
@@ -1478,12 +1509,14 @@ class ClusterEngine {
         // TODO: check that it works for both edges clustered
         //       (This might be paranoia)
       } else {
+        delete this._clusterEdges[edgeId]
+        this._restoreEdge(edge)
```
Comment (micahstubbs) [inline lib/network/modules/Clustering.js]: this is the meaningful part of the diff

**Comment by micahstubbs** [comment]
@Tyler-Maclachlan published to npm as `v4.24.10` 📦 🎉 
https://www.npmjs.com/package/visjs-network

**Comment by mojoaxel** [comment]
:love_letter: Thanks @Tyler-Maclachlan for your contribution!
This pull-request has been merge into https://github.com/visjs/vis-network/pull/30

**Pull request merged**